packages/vapor/scss/tables/table.scss

```diff
@@ -137,11 +137,23 @@
                 background-color: var(--white);
             }
         }
+
+        &.row-disabled {
+            td {
+                color: var(--grey-60);
+            }
+            > td:first-child {
+                border-left-color: var(--grey-60);
+            }
+            svg {
+                fill: var(--grey-60);
+            }
```

Comment:
Did you try to link with the admin (or just use a stroked icon) to see how it looks?